src/query/interpreter.cpp

```diff
@@ -2358,6 +2493,9 @@ Interpreter::PrepareResult Interpreter::Prepare(const std::string &query_string,
     } else if (utils::Downcast<IndexQuery>(parsed_query.query)) {
       prepared_query = PrepareIndexQuery(std::move(parsed_query), in_explicit_transaction_,
                                          &query_execution->notifications, interpreter_context_);
+    } else if (utils::Downcast<AnalyzeGraphQuery>(parsed_query.query)) {
```

Comment:
do we need both the db accessor and the storage accessor?

Reply:
True we don't, I'll refactor